opencog/nlp/types/atom_types.script

```diff
@@ -118,6 +118,7 @@ LG_CONN_DIR_NODE <- NODE    // e.g. "+"
 // Consists of the connector node, the direction, and possibly the
 // multi-connector node.
 LG_CONNECTOR <- LINK   // e.g. "MX+"
+MST_Connector <- LINK "MSTConnector"
 
```

Comment:
I am really sorry about that. I don't test my build every time I change something since I only need to test Scheme files. I apologize again. I'll correct the mistake immediately.